0.10/s2i/bin/run

```diff
@@ -8,7 +8,7 @@ run_node() {
   if [ "$DEV_MODE" == true ]; then
     exec nodemon --debug="$DEBUG_PORT"
   else
-    exec npm start -d
+    exec npm run $NPM_RUN
```

Comment:
Most people init their server using `npm start`, but `npm run start` is a valid alias. 

I need more info on the `-d` flag. I couldn't find any doc to cover it.  If it's needed, I could change the default config setting from `start` to `start -d`.

This customization would also allow additional init scripts to be added (to cover development modes - or for running `nodemon`, or `gulp`, or `grunt`).

Here are some examples of how that might look:

- http://substack.net/task_automation_with_npm_run
- https://docs.npmjs.com/cli/run-script
- https://docs.npmjs.com/misc/scripts#examples
- https://www.npmjs.com/package/scripty

This should init the example app using nodemon ([similarly to how "DEV_MODE" works](https://github.com/openshift/s2i-nodejs/blob/master/0.10/s2i/bin/run#L9)) by setting `NPM_RUN` to 'dev': https://github.com/openshift/s2ihttps://github.com/openshift/s2i-nodejs-nodejs/blob/master/0.10/test/test-app/package.json#L16 

This would allow developers to define multiple build or dev (or other runtime modes)